megamek/src/megamek/common/verifier/TestAdvancedAerospace.java

```diff
@@ -524,8 +511,9 @@ public String printWeightSail() {
     
     @Override
     public double getWeightFuel() {
-        // Add 2% for pumps and round up to the half ton
-        return ceil(vessel.getFuelTonnage() * 1.02, Ceil.TON);
+        // Fuel tanks and pumps add an additional 2%, rounded to the nearest ton
+        double pumpWeight = RoundWeight.nextTon(vessel.getFuelTonnage() * 0.02);
+        return vessel.getFuelTonnage() + pumpWeight;
```

Comment:
Why not just return RoundWeight.nextTon(vessel.getFuelTonnage() * 1.02) ?

Reply:
The tanks and pumps have to be rounded to the next ton, but the fuel itself can come in half ton lots.